docs/deployment/resource-providers/standalone/docker.zh.md

```diff
@@ -491,6 +491,27 @@ services:
         parallelism.default: 2
 ```
 
+### Enabling Python
+
+To build a custom image which has Python and Pyflink prepared, you can refer to the following Dockerfile:
+{% highlight Dockerfile %}
+FROM flink
+
+# install python3 and pip3
+RUN apt-get update -y && \
+apt-get install -y python3.7 python3-pip python3.7-dev && rm -rf /var/lib/apt/lists/*
```

Comment:
```suggestion
RUN apt-get update -y && \
    apt-get install -y python3.7 python3-pip python3.7-dev && \
    rm -rf /var/lib/apt/lists/*
```
I would reformat these lines to improve the readability.

Reply:
I would leave this untouched since I only want to preserve the current information.